### PR TITLE
Update OCP-4-13-release-notes for Developer Perspective: New Features

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
New Features and Enhancements Release Notes for Developer Perspective

Openshift 4.13 Release Notes

JIRA issue: [RHDEVDOCS-4867](https://issues.redhat.com/browse/RHDEVDOCS-4867)
Build Preview: https://57916--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-web-console

QE Review: @sanketpathak 
Peer Review: @bburt-rh

### CODE REVIEW

**Comment by sanketpathak** [comment]
Thanks @shreyasiddhartha, LGTM

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
 [id="ocp-4-13-web-console"]
 === Web console
+[id="ocp-4-13-developer-perspective"]
+==== Developer Perspective
+
+With this release, there are several updates to the *Developer* perspective of the web console. You can perform the following actions:
+
+* Create a *Serverless Function* through the *Import from Git* flow.
+* Create a *Serverless Function* through *Create Serverless Function* flow available on *Add page*.
+* The *Import from Git* workflow now supports *pipeline-as-code*.
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:130]:
```suggestion
* Use *pipeline-as-code* as an option in the *Import from Git* workflow.
```
The list item needs to have parallel syntactic structure with the other list items. My edit is one way to achieve this change, but feel free to try alternatives if my suggestion doesn't feel right to you.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
+* See which pods are receiving traffic in the:
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:131]:
```suggestion
* See which pods are receiving traffic in the following places:
```
See here for guidance regarding lead-in wording for lists.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
+** Side pane of the *Topology* view
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:132]:
```suggestion
** The side pane of the *Topology* view
```
Add definitel article

File: release_notes/ocp-4-13-release-notes.adoc [hunk 4]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
+** Pod *Details* view
+** *Pods* list view
+* Customize the timeout period or provide your own image when instantiating a *Web Terminal*.
+* Set default resources as an administrator to be pre-pinned in the *Developer* perspective navigation for all the users.
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:136]:
```suggestion
* Set default resources as an administrator to be pre-pinned in the *Developer* perspective navigation for all users.
```
Remove definite article

File: release_notes/ocp-4-13-release-notes.adoc [hunk 5]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
+
+[id="ocp-4-13-pipelines-page-improvements"]
+===== Pipelines page improvements
+In {product-title} 4.13, you can see the following navigation improvements on the *Pipelines* page:
+
+* The previously selected tab is visible when returning to the *Pipelines* page.
+* The default tab for *Repository details* page is now *PipelinesRuns*, but when you are following the *Create Git Repository* flow, the default tab is *Details*.
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:143]:
```suggestion
* The default tab for the *Repository details* page is now *PipelinesRuns*, but when you are following the *Create Git Repository* flow, the default tab is *Details*.
```
add definite article

File: release_notes/ocp-4-13-release-notes.adoc [hunk 6]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
+** Pod *Details* view
+** *Pods* list view
+* Customize the timeout period or provide your own image when instantiating a *Web Terminal*.
+* Set default resources as an administrator to be pre-pinned in the *Developer* perspective navigation for all the users.
+
+[id="ocp-4-13-pipelines-page-improvements"]
+===== Pipelines page improvements
+In {product-title} 4.13, you can see the following navigation improvements on the *Pipelines* page:
+
+* The previously selected tab is visible when returning to the *Pipelines* page.
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:142]:
```suggestion
* The tab you previously selected remains visible when you return to the *Pipelines* page.
```
- A couple of suggestions for improving user-centric wording

File: release_notes/ocp-4-13-release-notes.adoc [hunk 7]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
+* The default tab for *Repository details* page is now *PipelinesRuns*, but when you are following the *Create Git Repository* flow, the default tab is *Details*.
+
+[id="ocp-4-13-helm-page-improvements"]
+===== Helm page improvements
+In {product-title} 4.13, you can do the following from the *Helm* page:
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:147]:
```suggestion
In {product-title} 4.13, you can now do the following from the *Helm* page:
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 8]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
+
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:148]: The following list needs to be re-worded to make each item grammatically parallel: https://www.ibm.com/docs/en/ibm-style?topic=format-lists#wording-of-list-items

File: release_notes/ocp-4-13-release-notes.adoc [hunk 9]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
+* New terminology: Create and delete Helm releases instead of installing and uninstalling Helm charts.
+* Create and delete helm releases asynchronously and not wait for these actions to complete before performing the next task in the web console.
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:150]:
```suggestion
* You can create and delete Helm releases asynchronously and not wait for actions to complete before performing the next task in the web console.
```
I would suggest first updating the lead-in sentence, and then rewording this item to something similar to my suggestion.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 10]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
+* Helm release list now includes a *Status* column.
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:151]:
```suggestion
* The Helm release list now includes a *Status* column.
```
- Add definite article.
- Should `Helm` use an attribute?

File: release_notes/ocp-4-13-release-notes.adoc [hunk 11]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
 
 [id="ocp-4-13-web-console"]
 === Web console
+[id="ocp-4-13-developer-perspective"]
+==== Developer Perspective
+
+With this release, there are several updates to the *Developer* perspective of the web console. You can perform the following actions:
+
+* Create a *Serverless Function* through the *Import from Git* flow.
+* Create a *Serverless Function* through *Create Serverless Function* flow available on *Add page*.
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:129]:
```suggestion
* Create a *Serverless Function* by using the *Create Serverless Function* flow available on *Add page*.
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 12]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
 
 [id="ocp-4-13-web-console"]
 === Web console
+[id="ocp-4-13-developer-perspective"]
+==== Developer Perspective
+
+With this release, there are several updates to the *Developer* perspective of the web console. You can perform the following actions:
+
+* Create a *Serverless Function* through the *Import from Git* flow.
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:128]:
```suggestion
* Create a *Serverless Function* by using the *Import from Git* flow.
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 13]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
+* Create a *Serverless Function* through *Create Serverless Function* flow available on *Add page*.
+* The *Import from Git* workflow now supports *pipeline-as-code*.
+* See which pods are receiving traffic in the:
+** Side pane of the *Topology* view
+** Pod *Details* view
```
Comment (bburt-rh) [inline release_notes/ocp-4-13-release-notes.adoc:133]:
```suggestion
** The *Details* view for a pod
```

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:155]: Made this change. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc [hunk 14]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
 
 [id="ocp-4-13-web-console"]
 === Web console
+[id="ocp-4-13-developer-perspective"]
+==== Developer Perspective
+
+With this release, there are several updates to the *Developer* perspective of the web console. You can perform the following actions:
```
[the remark this answers is not in the document]
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:151]: Made this change. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc (hunk 3 above)
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:157]: Made this change. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc (hunk 4 above)
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:161]: Made this change. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc (hunk 6 above)
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:167]: Made this change. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc (hunk 7 above)
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:172]: This looks good. I'm using your version and making the change.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 15]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
+
+* Create a *Serverless Function* through the *Import from Git* flow.
+* Create a *Serverless Function* through *Create Serverless Function* flow available on *Add page*.
+* The *Import from Git* workflow now supports *pipeline-as-code*.
+* See which pods are receiving traffic in the:
+** Side pane of the *Topology* view
+** Pod *Details* view
+** *Pods* list view
```
[the remark this answers is not in the document]
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:159]: Made this change. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc (hunk 13 above)
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:158]: Made this change. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc (hunk 12 above)
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:153]: Noted and made this change. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc (hunk 11 above)
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:154]: Noted and made this change. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc [hunk 16]
```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
...
+
+[id="ocp-4-13-pipelines-page-improvements"]
+===== Pipelines page improvements
+In {product-title} 4.13, you can see the following navigation improvements on the *Pipelines* page:
+
+* The previously selected tab is visible when returning to the *Pipelines* page.
+* The default tab for *Repository details* page is now *PipelinesRuns*, but when you are following the *Create Git Repository* flow, the default tab is *Details*.
+
+[id="ocp-4-13-helm-page-improvements"]
+===== Helm page improvements
```
[the remark this answers is not in the document]
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:171]: If you look at our documentation, we have not used Helm as an attribute anywhere. I'm skeptical to change this. Skipping this suggestion for now.

File: release_notes/ocp-4-13-release-notes.adoc (hunk 8 above)
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:173]: I have followed your suggestions mentioned for the list and the lead-in sentence.

File: release_notes/ocp-4-13-release-notes.adoc (hunk 9 above)
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:175]: Made this change and as mentioned above I'm skipping the second suggestion.

File: release_notes/ocp-4-13-release-notes.adoc (hunk 10 above)
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc:176]: Made the first change and skipping the second suggestion.

File: release_notes/ocp-4-13-release-notes.adoc (hunk 16 above)
[the remark this answers is not in the document]
Comment (bburt-rh) [reply release_notes/ocp-4-13-release-notes.adoc:171]: Ok. Understood. Thanks!

**Review by bburt-rh — APPROVED** [review]

**Pull request merged**